Packs/FortinetFortiwebVM/Integrations/FortinetFortiwebVM/FortinetFortiwebVM.yml

```diff
@@ -2394,6 +2394,2519 @@ script:
+      isArray: false
+      name: backup_server
+      predefined:
+      - enable
+      - disable
+      required: false
+    deprecated: false
+    description: Update a rule for an reverse proxy server pool group. Supports API versions 1 & 2. Server pool rule advanced SSL settings dose not supported in this command.
+    execution: false
+    name: fortiwebvm-server-pool-reverse-proxy-rule-update
+  - arguments:
+    - description: Server pool group name (dependencies - use fortiwebvm-server-pool-group-list command to get all server pool groups).
+      isArray: false
+      name: group_name
+      required: true
+    - auto: PREDEFINED
+      defaultValue: enable
+      description: Server pool rule status.
+      isArray: false
+      name: status
+      predefined:
+      - enable
+      - disable
+      required: false
+    - description: The IP address of the web server to include in the pool.
+      isArray: false
+      name: ip
+      required: true
+    - defaultValue: '80'
+      description: "The TCP port number where the pool member listens for connections. The valid range is 1–65,535."
+      isArray: false
+      name: port
+      required: false
+    - auto: PREDEFINED
+      defaultValue: disable
+      description: Enable to use SSL/TLS for connections between FortiWeb and the pool member.
+      isArray: false
+      name: enable_ssl
+      predefined:
+      - enable
+      - disable
+      required: false
+    - description: "The name of the certificate that FortiWeb uses to decrypt SSL-secured connections. Requried when ssl=enable."
+      isArray: false
+      name: certificate_file
+      required: false
+    - auto: PREDEFINED
+      description: Enable to use a Server Name Indication (SNI) certificate. Server Name Indication allows multiple HTTPS websites to be served by the same IP address without requiring all those sites to use the same certificate. Supports by API V2 only.
+      isArray: false
+      name: enable_sni
+      predefined:
+      - enable
+      - disable
+      required: false
+    - description: The name of the Server Name Indication (SNI) certificate that specifies which certificate FortiWeb uses when encrypting or decrypting SSL-secured connections for a specified domain. (dependencies - use fortiwebvm-sni-certificate-list command to get all SNI certificates). Required when enable_sni=enable. Supports by API V2 only.
+      isArray: false
+      name: sni_certificate
+      required: false
+    deprecated: false
+    description: "Create a rule for an offline protection server pool group (server pool rule is a member of server pool group). Offline Protection  -  Requests are destined for a real web server instead of the FortiWeb appliance; traffic is duplicated to the FortiWeb through a span port. Supports API versions 1 & 2."
```

Comment:
```suggestion
    description: "Create a rule for an offline protection server pool group (server pool rule is a member of server pool group). Offline Protection - Requests are destined for a real web server instead of the FortiWeb appliance; traffic is duplicated to the FortiWeb through a span port. Supports API versions 1 & 2."
```